docs/privacy/reviews/sprint0_foundation.md

```
@@ -0,0 +1,111 @@
+# DSGVO-Review: Sprint 0 – Foundation
+
+**Reviewer:** Warp Agent  
+**Datum:** 2025-10-23  
+**Sprint:** S0 – Projekt-Fundament  
+**Status:** ✅ Bestanden
+
+---
+
+## Scope
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)